packages/core/src/dialects/sqlite/query-interface.js

```diff
@@ -74,13 +74,28 @@ export class SqliteQueryInterface extends AbstractQueryInterface {
     }
   }
 
+  /**
+   * @override
+   */
+  async showConstraint(tableName, constraintName, options) {
```

Comment:
could this method be moved to query-interface-typescript?

Reply:
Do you want me to create a `query-interface-typescript.ts` file for sqlite as all the other dialects support filtering constraints in the sql query?
Or should I include this the filter logic in the `AbstractQueryInterface`?

Reply:
A query-interface-typescript for sqlite :)

Reply:
This has been added now.
I also move the other constraint methods across to TS.